python/iceberg/core/manifest_reader.py

```diff
@@ -17,7 +17,7 @@
 
 import logging
 
-import fastavro
+import fastavro  # type: ignore
```

Comment:
Is this going to be needed for every import that doesn't use type annotations?

Reply:
Yes for now, or we can disable it. More and more packages will have types in the future :)